components/form-input/TotalSumForm.jsx

```diff
@@ -28,7 +28,14 @@ export const CounterButton = ({
   disableIncrement,
 }) => (
   <Box display="flex">
-    <Box mr={1} width="100%">
+    <Box
+      mr={2}
+      width="100%"
+      maxWidth="160px"
+      textAlign="right"
```

Comment:
I don't like the right aligned text I think its hard to read :(

I already fixed the button alignment so things stay in grids so I think we can go without this

Reply:
I saw you fixed this in your pr so I'll just merge this in and merge yours after!